src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -1161,7 +1202,7 @@ function WebRTCAdaptor(initialValues)
 		this.isConnected = function() {
 			return connected;
 		}
-
+		
 		wsConn.onmessage = function(event) {
 			var obj = JSON.parse(event.data);
 
```

Comment:
We don't have any sample data channel demo?

Reply:
By using browser console it can be used as:
webRTCAdaptor.sendViaDataChannel("streamId", "Message")

Reply:
we can also create a demo html within a new branch